client/src/App.js

```diff
@@ -6,6 +6,8 @@ import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
 import S3Test from "./pages/S3Test";
 import CreateProduct from "./pages/CreateProduct";
+import Register from "./pages/Register";
+import Login from "./pages/Login";
```

Comment:
Problem with loading this file because the file is called `login.jsx`

I suggest renaming that file to `Login.js` to stay consistent with either using js extention or jsx